runconfig/hostconfig.go

```diff
@@ -176,6 +180,7 @@ func ContainerHostConfigFromJob(job *engine.Job) *HostConfig {
 		Memory:          job.GetenvInt64("Memory"),
 		MemorySwap:      job.GetenvInt64("MemorySwap"),
 		CpuShares:       job.GetenvInt64("CpuShares"),
+		CpuPeriod:       job.GetenvInt64("CpuPeriod"),
```

Comment:
Hi,

As I commented in https://github.com/docker/docker/pull/11034#discussion-diff-27010644, we don't actually need any change in `ContainerHostConfigFromJob()`.